src/main/resources/org/jenkinsci/plugins/github/config/GitHubPluginConfig/config.groovy

```diff
@@ -4,6 +4,7 @@ import com.cloudbees.jenkins.GitHubPushTrigger
 import lib.FormTagLib
 
 def f = namespace(FormTagLib);
+def g = namespace("/lib/matrixauth")
```

Comment:
why matrixauth?

Reply:
```suggestion
def g = namespace("/lib/github")
```

Reply:
oops, i assume it worked because matrix auth is normally installed